src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -23,6 +24,8 @@ public class ModelManager implements Model {
     private final UserPrefs userPrefs;
     private final FilteredList<Member> filteredMembers;
 
```

Comment:
nit but should we remove this line spacing

Reply:
Roger, will fix that!